crates/evm-prover/src/grpc/server.rs

```diff
@@ -17,7 +17,8 @@ pub async fn create_grpc_server(config: Config) -> Result<()> {
         .build()
         .unwrap();
 
-    let prover_serivce = ProverService::default();
+    // TODO: Extend the config and plumb in here
+    let prover_serivce = ProverService::new()?;
```

Comment:
I'll create an issue to extend the server config type with fields necessary and then plumb them through in a follow up PR.

Essentially need things like:
- Namespace
- Rpc urls
- Path to evm genesis file

Reply:
https://github.com/celestiaorg/celestia-zkevm-hl-testnet/issues/80